collector/exec_bsd.go

```diff
@@ -72,6 +75,12 @@ func NewExecCollector() (Collector, error) {
 				description: "Number of fork() calls since system boot.  Resets at architeture unsigned integer.",
 				mib:         "vm.stats.vm.v_forks",
 			},
+			{
+				name:        "boot_time",
```

Comment:
Oh, yeah, I guess we should remove the exec part at some point. I think we only want to keep the sub namespace where it makes semantically sense. Though, given that's not fleshed out yet, I guess I won't change it further for now.